setup.py

```diff
@@ -155,7 +155,7 @@ def write_version(filename=os.path.join(*['airflow',
 postgres = ['psycopg2-binary>=2.7.4']
 ssh = ['paramiko>=2.1.1', 'pysftp>=0.2.9']
 salesforce = ['simple-salesforce>=0.72']
-s3 = ['boto3>=1.4.0']
+s3 = ['boto3>=1.7.0']
```

Comment:
Did you check in your own environment if this breaks anything?

Reply:
I've been running Airflow with boto3 1.7.4 in a few days and didn't see any problem so far (my environment is small and not for production though...)